update_graph_objs.py

```diff
@@ -0,0 +1,271 @@
+from plotly.graph_objs import graph_objs_tools
+from plotly.graph_reference import ARRAYS, CLASSES
+
+FLAG = '# AUTO-GENERATED BELOW. DO NOT EDIT! See makefile.'
+
+
+def get_non_generated_file_lines():
+    """
+    Copy each line up to our special FLAG line and return.
+
+    :raises: (ValueError) If the FLAG isn't found.
+    :return: (list) The lines we're copying.
+
+    """
+    lines_to_copy = []
+    flag_found = False
+    with open('./plotly/graph_objs/graph_objs.py', 'r') as f:
+        for line_to_copy in f:
+            if line_to_copy.startswith(FLAG):
+                flag_found = True
+                break
+            lines_to_copy.append(line_to_copy)
+    if not flag_found:
+        raise ValueError(
+            'Failed to find flag:\n''{}\n''in graph_objs_tools.py.'
+            .format(FLAG)
```

Comment:
More a suggestion, but for legibility, what do you think about putting the `format(FLAG)` on the line above? I think it's 80 chars on that line, but would be worth IMO